test/fcoe_client_write_spec.rb

```diff
@@ -52,34 +57,25 @@
       end
 
       before do
-        allow(Yast::SCR).to receive(:Write).and_return(true)
         allow(Yast::FileUtils).to receive(:Exists).and_return(false)
       end
 
-      it "smokes not" do
-        expect { subject.WriteSysconfigFiles }.to_not raise_error
-      end
-
-      it "writes the sysconfig configuration for the interface and its FCoE VLAN" do
-        expect(Yast::SCR).to receive(:Write)
-          .with(path_matching(/^\.network\.value\.\"eth1.500-fcoe\"\.*/), anything)
-        expect(Yast::SCR).to receive(:Write)
-          .with(path_matching(/^\.network\.value\.\"eth1\"\.*/), anything)
-        # A final call is also needed to flush the content
-        expect(Yast::SCR).to receive(:Write).with(Yast::Path.new(".network"), nil)
+      it "calls the FCoE connection generator to add or update the device and VLAN connections" do
+        expect_any_instance_of(Y2Network::FcoeConnGenerator)
+          .to receive(:update_connections_for).once.with(interfaces[1])
 
         subject.WriteSysconfigFiles
       end
 
-      it "writes nothing in /etc/sysconfig/network for interfaces without FCoE VLAN" do
-        allow(Yast::SCR).to receive(:Write) do |path, value|
-          # All calls to SCR.Write must contain a path starting with ".network.value.\"eth1"
-          # or exactly the path ".network" (for flushing)
-          expect(path.to_s).to match("(^\.network$)|(^\.network\.value\.\"eth1(\.|\"))")
-        end
+      it "writes the modified network connections configuration" do
+        expect(Yast::Lan).to receive(:write_config).with(only: [:connections])
 
         subject.WriteSysconfigFiles
       end
+
+      it "smokes not" do
```

Comment:
NP: I know this is as it was, but I'd change it for a something more meaningful. What about _does not raise an error_?